daskernetes/core.py

```diff
@@ -55,27 +52,29 @@ class KubeCluster(object):
     """
     def __init__(
             self,
+            worker_pod_template,
```

Comment:
It might be nice to accept this as either a `kubernetes` library object, a dictionary, or a filename, and then normalize all options into a kubernetes object.

Reply:
I made individual classmethods for these variants instead. I'm generally not a fan of doing polymorphism style stuff in Python constructors, since I thinks this is clearer.